app/public/static/js/components/ordered-puzzle.js

```diff
@@ -84,9 +84,16 @@ AFRAME.registerComponent("ordered-puzzle", {
         const puzzlePiece = puzzlePieceCache[index];
 
         // Calculate the puzzle piece width and height based on image ratio
-        const ratio = rawImageEl.width / rawImageEl.height;
+        let ratio = rawImageEl.height / rawImageEl.width;
```

Comment:
This should stay as `rawImageEl.width / rawImageEl.height;` right? Since for non-jigsaw, we want the current behaviour.

Reply:
I think currently, even for non-jigsaw the issue of squishing seems to exist. When i changed the ordered puzzle to use a narrow picture, you can see the effect is still there
![image](https://user-images.githubusercontent.com/35475783/126724325-b8412c6c-8111-409f-8a23-5a086012dc93.png)

Reply:
Gotcha, okay, code perspective, it looks good to me. @amoliksingh can you test out with the images you were trying with the ordered puzzle and make sure it still spawns the visual targets okay?